### DISCUSSION TITLE
ABB IRB460

### DISCUSSION THREAD

**Discussion by francescobic** [visose/Robots#96] [Q&A] [answered]
Hi 

is possible to load in the library a 4 axis robot? ABB IRB460 
i've try to follow the structure for 6 axis robot but when i put  "load robot System" component and select my 4 axis robot the component show me the message :  indices over matrix limit

best

**Comment by visose** [comment, marked as answer]
There's no support for 4 axis robots. You might be able to 'hack' one in, but I'm not sure exactly if it would be possible as it is, without writing some code.
